File: backend/portfolio-backend/src/main/java/com/mattysleduc/portfolio/domain/skillsubdomain/presentation_layer/SkillResponseModel.java
package com.mattysleduc.portfolio.domain.skillsubdomain.presentation_layer;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class SkillResponseModel {
    private String skillId;
    private String name;
    private String description;
    private String category;
    private Integer level;
}

File: frontend/src/features/skills/models/SkillResponseModel.ts
export interface SkillResponseModel {
  [key: string]: any;
  skillId: string;
  name: string;
  nameEn?: string;
  nameFr?: string;
  description: string;
  descriptionEn?: string;
  descriptionFr?: string;
  category?: string;
  level?: number;
}
